.github/workflows/configlet.yml

```diff
@@ -26,6 +26,8 @@ jobs:
 
       - name: Fetch configlet
         uses: exercism/github-actions/configlet-ci@main
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
Docs tell me this should be `GITHUB_TOKEN`: https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows but I was also under the impression that `GITHUB_TOKEN` is already set by default and is only needed if one wants to use a non-default token.

Since you tested it, I assume it works, just pointing this out so we can be sure. Feel free to ignore my comment if you've already checked it.

We could test if it works as intended by temporarily adding a call to the rate limit endpoint for debugging:

```
gh api \
  -H "Accept: application/vnd.github+json" \
  /rate_limit
```

which should return something just under 1000 requests remaining if the authentication worked properly.

Reply:
> That's what I thought too. But when I tested the second commit in this PR, [CI failed with](https://github.com/ee7/exercism-nim/actions/runs/3856652767/jobs/6573121258#step:3:11):

Love their consistency 🥴 

Anyway, in that case consider my comment moot

Reply:
I noticed the `github.token` reference too. 

> Nice idea. I'll try it later. But I'm led to believe that gh release download never makes an unauthenticated request, and fails immediately otherwise.

Interesting. One way in which we could check that would be to check the rate limit before and after downloading the release. This can be done via `gh api /rate_limit`. @ee7 Could you try that?

Reply:
Perfect!